src/Test.ConcurrencyTests/Test.ConcurrencyTests.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
not sure where it is set, I see our test has both 3.1 and 6.0
![image](https://user-images.githubusercontent.com/81775155/233218928-5837558b-4a3e-42d1-be65-d65558067106.png)

Reply:
@shaopeng-gh I deliberately left it to netcoreapp3.1 because Coyote does not support net6.0 yet.

Reply:
got it.